libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-lookup/cell-editor-lookup.component.html

```diff
@@ -44,7 +43,6 @@
     [data]="skyComponentProperties?.data"
     [debounceTime]="skyComponentProperties?.debounceTime"
     [descriptorProperty]="skyComponentProperties?.descriptorProperty"
-    [disabled]="skyComponentProperties?.disabled || false"
```

Comment:
why remove this?

Reply:
This is a reactive form so it should not be passing this input. It's handling the form control enable/disable within the component `.ts` file.